src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Ssl.cs

```diff
@@ -69,10 +69,10 @@ internal static partial class Ssl
             return result;
         }
 
-        [DllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_SslWrite")]
+        [DllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_SslWrite", SetLastError = true)]
```

Comment:
Based on the call sites to BioWrite, I was about to say we also need SetLastError = true on it, but then from looking at the helper, it looks like there's actually dead code / branches at call sites to BioWrite, because BioWrite throws if the number of bytes written doesn't match the number of bytes requested to be written.  So these branches would appear to all be dead:
- Will never happen: https://github.com/dotnet/runtime/blob/8a52f1e948b6f22f418817ec1068f07b8dae2aa5/src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs#L238-L241
- Will always happen: https://github.com/dotnet/runtime/blob/8a52f1e948b6f22f418817ec1068f07b8dae2aa5/src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs#L362